src/app/user/_tests_/user.spec.js

```diff
@@ -15,7 +15,7 @@ const userRequestObject = {
   role: 'admin',
 };
 
-describe('User Test Suite', () => {
+describe.only('User Test Suite', () => {
```

Comment:
Hi, you forgot to remove `only` from this block.

Reply:
Oops, I'll remove it.